<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxPlotDialog</class>
 <widget class="QDialog" name="BoxPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>583</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="bt_close">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_layer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_field">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>321</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Box Plot</string>
   </property>
   <widget class="BoxPlotDraw" name="BoxPlotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>300</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_draw">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoxPlotDraw</class>
   <extends>QWidget</extends>
   <header location="global">boxplotdraw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
